CMakeLists.txt

```diff
@@ -115,7 +117,6 @@ if (LIBMEM_BUILD_TESTS)
 endif()
 
 set_target_properties(libmem PROPERTIES POSITION_INDEPENDENT_CODE True INCLUDES ${LIBMEM_INC})
```

Comment:
I don't understand these suggestions. Don't we want to add LM_STATIC only if "LIBMEM_BUILD_STATIC" is true?

Reply:
For me, the ideal would be, for any OS:
StaticBuild: LM_STATIC
DynamicBuild: LM_EXPORT

It's also fine if we have LM_EXPORT all the time, like so:
StaticBuild: LM_STATIC and LM_EXPORT
DynamicBuild: LM_EXPORT

But the changes you suggested make it like this:
StaticBuild: LM_STATIC and LM_EXPORT
DynamicBuild: LM_STATIC and LM_EXPORT
So that it never adds the "__declspec(dllexport)" and so a dynamic library is never produced.

Or am I misunderstanding the changes?